okteto.yml

```diff
@@ -0,0 +1,18 @@
+name: hello-world
+build:
+  hello-world:
+    context: .
+    dockerfile: Dockerfile
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/dev:latest
```

Comment:
remove `image`. for nodejs applications, we reuse the same image as the original deployment

Reply:
You didn't remove the `image` field